lib/tasks/user.rake

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+namespace :user do
+  desc "Migrate wrong age"
```

Comment:
@hengsokly please kindly use the appropriate term, in this case I think, use "Migrate unicode age" rather than "Migrate wrong age"